site/src/components/scroll.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Scroll
+description: Scroll related components in the Atlas Design System
+template: standard
+---
+
+# Scrolling
+
+There are two components that help modify the scroll direction of a particular container.
+
+## Horizontal scrolling
+
+Force an elements overflow to be horizontally scrollable by using `.scroll-horizontal`.
+
+<div class="scroll-horizontal border background-color-success padding-xl margin-top-md">
+	<div class="">
```

Comment:
Is it also needed? Sorry missed those initially 🤦🏻‍♀️

Reply:
updated!